### PR TITLE
Class: add property blabla2.

### PR DESCRIPTION
**Pull request by michaellightricks**

### CODE REVIEW

File: Class.h [hunk 1]
```diff
@@ -7,4 +7,6 @@
 // this property is ble ble
 @property (nonatomic) BOOL bleble;
 
+@property (nonatomic) BOOL blabla2;
```
Comment (xorserer) [inline Class.h]: Add docs.

**Comment by xorserer** [comment]
Round 1.

**Comment by michaellightricks** [comment]
Round done.

**Comment by xorserer** [comment]
Squash & rebase.

**Comment by michaellightricks** [comment]
Done.

**Pull request merged**